src/pygama/dsp/build_dsp.py

```diff
@@ -128,36 +130,37 @@ def build_dsp(f_raw, f_dsp, dsp_config, lh5_tables=None, database=None,
     for tb in lh5_tables:
         # load primary table and build processing chain and output table
         tot_n_rows = raw_store.read_n_rows(tb, f_raw)
-        if n_max and n_max<tot_n_rows: tot_n_rows=n_max
+        if n_max and n_max < tot_n_rows:
+            tot_n_rows = n_max
 
         # if we have separate DSP files for each table, read them in here
         if chan_config is not None:
             f_config = chan_config[tb]
             with open(f_config) as config_file:
-                dsp_config = json.load(config_file, object_pairs_hook=OrderedDict)
-            print('Processing table:', tb, 'with DSP config file:\n  ', f_config)
+                dsp_config = json.load(config_file, object_pairs_hook=json.OrderedDict)
```

Comment:
object_pairs_hook is not needed; order does not matter
```suggestion
                dsp_config = json.load(config_file)
```